simulation/schema.yaml

```
@@ -0,0 +1,18 @@
+version: string()
+
+server:
+  network: enum('regtest', 'testnet', 'mainnet')
+  initial_funding: float(min=0)
+  round_interval: int(min=1)
+
+clients:
+  - id: string()
+    initial_funding: float(min=0, required=false)
+
+rounds:
+  - number: int(min=1)
+    actions:
+      regex('^client_\d+$'):
+        - type: enum('Onboard', 'SendAsync', 'SendOffchain', 'SendOnchain', 'Claim', 'CollaborativeExit', 'UnilateralExit')
```

In order to maintain coherence with method's names, I think "CollaborativeExit" should be renamed to "CollaborativeRedeem"

```suggestion
        - type: enum('Onboard', 'SendAsync', 'SendOffchain', 'SendOnchain', 'Claim', 'CollaborativeRedeem', 'UnilateralExit')
```

---

but redeem can be confused with Redeem Tx for oor?